applications/fishing-map/src/features/sidebar/context-areas/ContextAreaSection.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { selectContextAreasDataviews } from 'features/workspace/workspace.selectors'
+import styles from 'features/sidebar/common/Sections.module.css'
+import LayerPanel from './ContextAreaLayerPanel'
+
+function VesselsSection(): React.ReactElement {
```

Comment:
nitpick
```suggestion
export default function(): React.ReactElement {
```

Reply:
I prefer having the name so in the Component debugger tree it is easily to debug rathern than `anonymous function`

Reply:
I was bothered by the fact it's called 'Vessels'